<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QWidget" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>765</width>
    <height>450</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="windowTitle">
   <string>登录</string>
  </property>
  <property name="styleSheet">
   <string notr="true">

QPushButton[minimize=true]{
border-image: url(:/images/min.png);
}
QPushButton[minimize=true]:hover{
border-image: url(:/images/min_hover.png);
}
QPushButton[minimize=true]:pressed{
border-image: url(:/images/min_pressed.png);
}

QPushButton[close=true]{
border-image: url(:/images/close.png);
}
QPushButton[close=true]:hover{
border-image: url(:/images/close_hover.png);
}
QPushButton[close=true]:pressed{
border-image: url(:/images/close_pressed.png);
}

QPushButton[common=true]{
border:1px solid  rgb(154, 154, 154);
	background-color: qlineargradient(spread:pad, x1:0.5, y1:1, x2:0.5, y2:0, stop:0.00568182 rgba(225, 225, 225, 255), stop:0.5 rgba(236, 236, 236, 255), stop:1 rgba(225, 225, 225, 255));
border-radius:1px;
padding:5px;
}
QPushButton[common=true]:hover{
border:1px solid  rgb(0, 170, 255);
	background-color: qlineargradient(spread:pad, x1:0.5, y1:1, x2:0.5, y2:0, stop:0.00568182 rgba(235, 235, 235, 255), stop:0.5 rgba(246, 246, 246, 255), stop:1 rgba(235, 235, 235, 255));
border-radius:1px;
padding:5px;
}

QPushButton[common=true]:pressed{
border:1px solid  rgb(200, 200, 200);
	background-color: qlineargradient(spread:pad, x1:0.5, y1:1, x2:0.5, y2:0, stop:0.00568182 rgba(210, 210, 210, 255), stop:0.5 rgba(205, 205, 205, 205), stop:1 rgba(210, 210, 210, 255));
border-radius:1px;
padding:5px;
}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget_parent" native="true">
     <property name="styleSheet">
      <string notr="true">QWidget#widget_parent{
/*border-image: url(:/images/login_bg.png);*/
background-color: rgb(0, 170, 255);
}</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>20</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>5</number>
        </property>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_close">
          <property name="minimumSize">
           <size>
            <width>27</width>
            <height>22</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>27</width>
            <height>22</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
border-image: url(:/images/close.png);
}
QPushButton:hover{
border-image: url(:/images/close_hover.png);
}
QPushButton:pressed{
border-image: url(:/images/close_pressed.png);
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_5">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QStackedWidget" name="stackedWidget">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="page">
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <property name="leftMargin">
             <number>10</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>10</number>
            </property>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QWidget" name="widget_login" native="true">
              <property name="styleSheet">
               <string notr="true">QWidget#widget_login{
background-color: rgb(250, 250, 250);
border:1px solid rgb(235, 235, 235);
border-radius:0px;
}</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_2">
               <property name="leftMargin">
                <number>20</number>
               </property>
               <property name="topMargin">
                <number>20</number>
               </property>
               <property name="rightMargin">
                <number>25</number>
               </property>
               <property name="bottomMargin">
                <number>30</number>
               </property>
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_8">
                 <property name="bottomMargin">
                  <number>45</number>
                 </property>
                 <item>
                  <widget class="QLabel" name="label_5">
                   <property name="font">
                    <font>
                     <family>微软雅黑</family>
                     <pointsize>18</pointsize>
                     <weight>75</weight>
                     <bold>true</bold>
                    </font>
                   </property>
                   <property name="text">
                    <string>机动车尾气监测系统</string>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignCenter</set>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_6">
                   <property name="maximumSize">
                    <size>
                     <width>16777215</width>
                     <height>5</height>
                    </size>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">border:1px;
border-bottom:1px solid gray;</string>
                   </property>
                   <property name="text">
                    <string/>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QGridLayout" name="gridLayout">
                 <property name="leftMargin">
                  <number>10</number>
                 </property>
                 <property name="rightMargin">
                  <number>10</number>
                 </property>
                 <property name="horizontalSpacing">
                  <number>8</number>
                 </property>
                 <property name="verticalSpacing">
                  <number>30</number>
                 </property>
                 <item row="0" column="0">
                  <widget class="QLabel" name="label">
                   <property name="font">
                    <font>
                     <family>微软雅黑</family>
                    </font>
                   </property>
                   <property name="text">
                    <string>用户名</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="0">
                  <widget class="QLabel" name="label_2">
                   <property name="font">
                    <font>
                     <family>微软雅黑</family>
                    </font>
                   </property>
                   <property name="text">
                    <string>密  码</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <widget class="QLineEdit" name="lineEdit_user_name">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="minimumSize">
                    <size>
                     <width>160</width>
                     <height>25</height>
                    </size>
                   </property>
                   <property name="maxLength">
                    <number>28</number>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="1">
                  <widget class="QLineEdit" name="lineEdit_user_password">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="minimumSize">
                    <size>
                     <width>0</width>
                     <height>25</height>
                    </size>
                   </property>
                   <property name="inputMask">
                    <string/>
                   </property>
                   <property name="text">
                    <string/>
                   </property>
                   <property name="maxLength">
                    <number>32767</number>
                   </property>
                   <property name="echoMode">
                    <enum>QLineEdit::Password</enum>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="1">
                  <widget class="QPushButton" name="pushButton_login">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="font">
                    <font>
                     <family>微软雅黑</family>
                    </font>
                   </property>
                   <property name="focusPolicy">
                    <enum>Qt::NoFocus</enum>
                   </property>
                   <property name="text">
                    <string>登录</string>
                   </property>
                   <property name="common" stdset="0">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page_2">
         <layout class="QVBoxLayout" name="verticalLayout_6"/>
        </widget>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_6">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
